src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -205,51 +237,219 @@ const RadioWithTippy = (children, isFromCDPipeline: boolean, tippyContent: strin
-          </ConditionalWrap>
-          <ConditionalWrap
-              condition={allowedDeploymentTypes.indexOf(DeploymentAppTypes.GITOPS) === -1}
-              wrap={(children) =>
-                  RadioWithTippy(children, isFromCDPipeline, 'Deployment to this environment is not allowed via GitOps')
-              }
-          >
-              <RadioGroupItem
-                  dataTestId="gitops-deployment"
-                  value={DeploymentAppTypes.GITOPS}
-                  disabled={allowedDeploymentTypes.indexOf(DeploymentAppTypes.GITOPS) === -1}
-              >
-                  GitOps
-              </RadioGroupItem>
-          </ConditionalWrap>
-      </RadioGroup>
-  )
+    const gitOpsNotCongiguredText =
+        allowedDeploymentTypes.length == 1 ? GITOPS_REPO_REQUIRED_FOR_ENV : GITOPS_REPO_REQUIRED
+    return (
+        <>
+            <RadioGroup
+                value={deploymentAppType}
+                name="DeploymentAppTypeGroup"
+                onChange={handleOnChange}
+                disabled={isDisabled}
+                className={rootClassName ?? ''}
+            >
+                <ConditionalWrap
+                    condition={allowedDeploymentTypes.indexOf(DeploymentAppTypes.HELM) === -1}
+                    wrap={(children) =>
+                        RadioWithTippy(
+                            children,
+                            isFromCDPipeline,
+                            'Deployment to this environment is not allowed via Helm',
+                        )
+                    }
+                >
+                    <RadioGroupItem
+                        dataTestId="helm-deployment"
+                        value={DeploymentAppTypes.HELM}
+                        disabled={allowedDeploymentTypes.indexOf(DeploymentAppTypes.HELM) === -1}
+                    >
+                        Helm
+                    </RadioGroupItem>
+                </ConditionalWrap>
+                <ConditionalWrap
+                    condition={allowedDeploymentTypes.indexOf(DeploymentAppTypes.GITOPS) === -1}
+                    wrap={(children) =>
+                        RadioWithTippy(
+                            children,
+                            isFromCDPipeline,
+                            'Deployment to this environment is not allowed via GitOps',
+                        )
+                    }
+                >
+                    <RadioGroupItem
+                        dataTestId="gitops-deployment"
+                        value={DeploymentAppTypes.GITOPS}
+                        disabled={allowedDeploymentTypes.indexOf(DeploymentAppTypes.GITOPS) === -1}
+                    >
+                        GitOps
+                    </RadioGroupItem>
+                </ConditionalWrap>
+            </RadioGroup>
+            {deploymentAppType === DeploymentAppTypes.GITOPS &&
+                isGitOpsRepoNotConfigured &&
+                !window._env_.HIDE_GITOPS_OR_HELM_OPTION && (
```

Comment:
we can remove this `!window._env_.HIDE_GITOPS_OR_HELM_OPTION` as this component will only load in that case

Reply:
yes, done